### PR TITLE
feat: add grafana&prometheus

### PR DESCRIPTION
**Pull request by weichweich**

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -1,6 +1,6 @@
-KILT_IMG=kiltprotocol/peregrine:develop
-KILT_RAW_SPEC_FILE=/node/dev-specs/kilt-parachain/peregrine-kilt-fast-gov.json
-KILT_RUNTIME=mashnet
+KILT_IMG=kiltprotocol/peregrine:0.2.0-69f254e
+KILT_RAW_SPEC_FILE=spiritnet-dev
+KILT_RUNTIME=spiritnet
 
 POLKADOT_RAW_SPEC_FILE=polkadot.raw.json
-POLKADOT_IMG=parity/polkadot:v0.9.5
+POLKADOT_IMG=parity/polkadot:v0.9.7
```
Comment (rflechtner) [inline .env:6]: new relay chain? is that what we use on wilt?

Comment (weichweich) [reply .env:6]: shouldn't affect rococo but that's the latest wilt

**Review by rflechtner — APPROVED** [review]

File: docker-compose.yml [hunk 2]
```diff
@@ -123,17 +130,20 @@ services:
...
     expose:
       - 40333
       - 30333
+      - 9615
     depends_on:
       - relay-node-alice
       - relay-node-bob
       - relay-node-charlie
+      - prometheus
+      - grafana
```
[the remark this answers is not in the document]
Comment (weichweich) [reply docker-compose.yml:146]: Just wanted to make sure that all metrics are captured from the beginning. If you don't want grafana&prometheus then you need to remove everything related to them. 😓

File: monitoring/grafana-provisioning/dashboards/default.yaml [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+
+providers:
+  - name: 'Prometheus'
+    orgId: 1
+    folder: ''
+    type: file
+    disableDeletion: false
+    editable: true
+    options:
+      path: /etc/grafana/provisioning/dashboards
```
[the remark this answers is not in the document]
Comment (weichweich) [reply monitoring/grafana-provisioning/dashboards/default.yaml:11]: ah we do

**Comment by weichweich** [comment]
looks stale?

**Pull request closed without merging**